src/env.cc

```diff
@@ -380,6 +380,9 @@ Environment::Environment(IsolateData* isolate_data,
     async_hooks_.no_force_checks();
   }
 
+  if (options_->force_context_aware)
+    force_context_aware_ = true;
```

Comment:
Is there a particular reason for storing this? Using `env->options()->force_context_aware` should work just as well.